pkg/edgeview/src/basics.go

```diff
@@ -143,6 +143,56 @@ func initOpts() {
 	}
 }
 
+// this script is automatically installed into your /tmp/download/bin
+// and can be used to run your kubectl with the download kubeconfig file
+// decrypted by the ssh private key
+const kubeConfDecryptScript = `#!/bin/bash
+
+usage() {
+    echo "Usage: $0 [-keypath=\"your ssh private key file path\"]"
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    key="$1"
+
+    case $key in
+        -keypath=*)
+            keypath="${key#*=}"
+            shift # Shift to the next argument after the keypath value
+            ;;
+        *)
+            # Unknown option or argument
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+# Set the private key path based on the provided -keypath or use the default
+if [ -n "$keypath" ]; then
+    privateKeyFile="$keypath"
+else
+    # use default ssh private key
+    privateKeyFile="$HOME/.ssh/id_rsa"
+fi
+
+# the symmetric key is encrypted by ssh public key
+symmetricKeyEncFile="/tmp/download/kube-symmetric-file.enc"
+# the kubeconfig file is encrypted by the symmetric key
+symmetricEncFile="/tmp/download/kube-config-yaml"
+
+# Read the encrypted symmetric key from file
+encryptedSymKey=$(cat "$symmetricKeyEncFile")
```

Comment:
Where is `encryptedSymKey` used?

Reply:
maybe left over from change, let me remove.